git-fire

```diff
@@ -27,6 +27,7 @@ new_branch() {
 }
 
 fire() {
+	local initial_branch message remote
```

Comment:
Is `local` portable?

Reply:
quick googling gives info that posix has not standardized it yet:
- https://unix.stackexchange.com/a/319856/178169
- https://github.com/thoughtbot/laptop/issues/481
- https://stackoverflow.com/questions/18597697/posix-compliant-way-to-scope-variables-to-a-function-in-a-shell-script#answer-18600920

but it's present in shell i've used over two decades...